### DISCUSSION TITLE
Icon request: fa-diaper

### DISCUSSION THREAD

**Discussion by privatemiao** [FortAwesome/Font-Awesome#21248] [Icon Requests]
![image](https://cloud.githubusercontent.com/assets/1772357/5212056/96e16196-7629-11e4-9c49-dd79cde23c7b.png)

**Comment by phishyatl** [comment]
+1

**Comment by stgogm** [comment]
+1

**Comment by bhpatel26** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
